### PR TITLE
Fix: IPV4 can't display on zipkin server

### PR DESCRIPTION
**Pull request by anthonywanted**
fix the problem mentioned here
https://github.com/openzipkin/zipkin4net/issues/165

### CODE REVIEW

File: Src/zipkin4net/Src/Tracers/Zipkin/JSONSpanSerializer.cs [hunk 1]
```diff
@@ -133,6 +147,16 @@ private static void SerializeEndPoint(StreamWriter writer, IPEndPoint endPoint,
 
     delegate void SerializeMethod<T>(StreamWriter writer, T element, IPEndPoint endPoint, string serviceName);
 
+    public class JSONSpanSerializerOption
```
Comment (fedj) [inline Src/zipkin4net/Src/Tracers/Zipkin/JSONSpanSerializer.cs]: I'm not sure it's mandatory here. If you take a look at pkin/src/main/java/zipkin/internal/JsonCodec.java#L95, it's always serialized as a string, not as an int (which was clearly a bug)

**Review by fedj — CHANGES_REQUESTED** [review]

File: Src/zipkin4net/Src/Tracers/Zipkin/JSONSpanSerializer.cs (hunk 1 above)
Comment (anthonywanted) [reply Src/zipkin4net/Src/Tracers/Zipkin/JSONSpanSerializer.cs]: Oh. That’s great.....

**Comment by fedj** [comment]
Can you rebase or cherry pick the last commit on another branch? I can't merge it right now

**Comment by codefromthecrypt** [comment]
Thanks for sorting this!

**Pull request closed without merging**